.github/workflows/build-virtualenv-caches.yml

```diff
@@ -95,3 +95,37 @@ jobs:
       run: echo "PYVER=$(echo ${{ matrix.python-version}} |cut -d. -f1,2)" >> $GITHUB_ENV
     - run: |
         make benchenv_py${PYVER}
+
+  build_integration_virtualenv:
```

Comment:
We have the cache builds for the master branch because that way we get pre-cached virtualenvs for new PRs since caches are never shared between different PRs.

I didn't realize, but we can in fact share the same cache between integration and coverage, I'll adjust that.

Reply:
Ah, thanks. I missed that the cache build happens on master.